.palantir/revapi.yml

```diff
@@ -1,5 +1,5 @@
 versionOverrides:
-  org.apache.iceberg:iceberg-api:apache-iceberg-0.14.0: "0.14.0"
+  org.apache.iceberg:iceberg-api:apache-iceberg-1.0.0: "1.0.0"
```

Comment:
not sure why didn't we have 0.14.1 before and does that tag version here need to be from the master? Currently, `apache-iceberg-1.0.0` is not from the master.

Reply:
looks like no CI was executed with this change, can you add `revapi.yml` file to https://github.com/apache/iceberg/blob/master/.github/workflows/api-binary-compatibility.yml#L29 so that CI runs?

Reply:
Added. Thanks for the suggestion.

Reply:
I think this failed now because the tag is not from the master? Looks like it even tried 0.14.0 but not 0.14.1